packages/xxhash.rb

```diff
@@ -3,29 +3,22 @@
 class Xxhash < Package
   description 'xxHash is an extremely fast non-cryptographic hash algorithm, working at speeds close to RAM limits.'
   homepage 'https://cyan4973.github.io/xxHash/'
-  version '0.7.3'
+  version '0.8.0'
   compatibility 'all'
-  source_url 'https://github.com/Cyan4973/xxHash/archive/v0.7.3.tar.gz'
-  source_sha256 '952ebbf5b11fbf59ae5d760a562d1e9112278f244340ad7714e8556cbe54f7f7'
+  source_url 'https://github.com/Cyan4973/xxHash/archive/v0.8.0.tar.gz'
+  source_sha256 '7054c3ebd169c97b64a92d7b994ab63c70dd53a06974f1f630ab782c28db0f4f'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/xxhash-0.7.3-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/xxhash-0.7.3-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/xxhash-0.7.3-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/xxhash-0.7.3-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'ebc5da796fed9aa28b1d1dfea223be7b82cb2b10a089a3fe17da44a526ce6d7f',
-     armv7l: 'ebc5da796fed9aa28b1d1dfea223be7b82cb2b10a089a3fe17da44a526ce6d7f',
-       i686: '5c4d9bf621e94373d0453f06a0a1fbd8f10002ba24f53355580eb52316653875',
-     x86_64: 'af56f28e9ef99aefed260fb19f931f9ddbe2ec1bdd22a90138e66f76ccbbe203',
   })
 
   def self.build
     system 'make', "PREFIX=#{CREW_PREFIX}"
   end
 
   def self.install
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install' 
     FileUtils.mkdir_p ["#{CREW_DEST_PREFIX}/bin", "#{CREW_DEST_MAN_PREFIX}/man1"]
     FileUtils.cp 'xxhsum', "#{CREW_DEST_PREFIX}/bin/"
     FileUtils.cp 'xxhsum.1', "#{CREW_DEST_MAN_PREFIX}/man1/"
```

Comment:
Does the `system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'` line negate the `FileUtils...` commands that follow?

Reply:
Indeed FileUtils lines are useless now. I removed them